src/main/java/com/palantir/gradle/gitversion/NativeGitDescribe.java

```diff
@@ -1,126 +0,0 @@
-/*
```

Comment:
out of curiosity, what's the reason for removing this? If there's a potential future where we remove jgit, don't we want to keep this code around?

Reply:
> because it won't be used if we do not call Native git.

not sure i follow - don't we _always_ try native git first, and then fall back to jgit if it fails?